roles/neutron-data/tasks/main.yml

```diff
@@ -28,12 +32,20 @@
   service: name=neutron-openvswitch-agent state=started
   when: neutron.plugin == 'ovs'
 
+- name: ml2 dataplane config
```

Comment:
in environments where the network node does not live on the controller, we need to split up the ml2 configuration.